scripts/rotate-aws-creds/sharded.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+set -euo pipefail
+
+source "$(dirname $0)/common.sh"
+
+SHARD_SSH="ssh root@hive-production-master"
+OC_CMD="oc --insecure-skip-tls-verify=true"
+
```

Comment:
Should these go to common?

Reply:
Also `SHARD_SSH` should be `SHARD_PROD_SSH` do these script support multiple ENV's?

Reply:
They support the both staging and production because the production hive(`hive-production`) is used to create both the staging and production sharded hives.

Reply:
Also these variable are only used when accessing sharded hives so they should stay here IMO.